datasets/datasets.py

```diff
@@ -96,7 +99,13 @@ def get_dataset(name: str) -> Dict[str, Any]:
             columns = metadata["columns"]
             featuretypes = metadata["featuretypes"]
             columns = [{"name": col, "featuretype": ftype} for col, ftype in zip(columns, featuretypes)]
-            return {"name": name, "columns": columns, "filename": filename}
+            content = load_dataset(name=filename)
+            data = content.values.tolist()
```

Comment:
@samborba será necessário fazer um "nan to null". O retorno do flask é um JSON inválido (com NaNs).
![nans](https://user-images.githubusercontent.com/1791381/92527094-dbce3b80-f1fc-11ea-851a-b4da706458da.PNG)